app/lib/server/startup/email.js

```diff
@@ -288,6 +288,11 @@ settings.addGroup('Email', function() {
 			placeholder: 'email@domain',
 			secret: true,
 		});
+		this.add('Direct_Reply_ReplyTo_Enable', false, {
+			type: 'boolean',
+			env: true,
+			i18nLabel: 'Add Reply-To header',
```

Comment:
This label should follow the convention for other labels, which is a "capitalized" snake case. 
```suggestion
			i18nLabel: 'Add_ReplyTo_Header',
```